src/random_lcg.cpp

```diff
@@ -17,7 +17,8 @@ constexpr uint64_t prn_stride {152917LL}; // stride between particles
 //==============================================================================
 
 // 64 bit implementation of the PCG-RXS-M-XS 64-bit state / 64-bit output
-// geneator Adapted from: https://github.com/imneme/pcg-c
+// geneator Adapted from: https://github.com/imneme/pcg-c, in particular
+// https://github.com/imneme/pcg-c/blob/83252d9c23df9c82ecb42210afed61a7b42402d7/include/pcg_variants.h#L188-L192
```

Comment:
I added a more detailed reference to the code. It's not related to the issue, but I thought it might be useful anyway.